iocore/hostdb/P_RefCountCache.h

```diff
@@ -154,7 +154,7 @@ template <class C> class RefCountCachePartition
   template <class Iterator> void dealloc_entry(Iterator ptr);
 
   size_t count() const;
-  void copy(Vec<RefCountCacheHashEntry *> &items);
+  void copy(std::vector<RefCountCacheHashEntry *> &items);
```

Comment:
`const&`

Reply:
I do not get where do you indent to add const& qualifier here? Into method's declaration/definition? If so it causes compilation error.

Reply:
Oh Sweet Turing, it's a copy *to*, not a copy. It copies from `this` to `items`. Nevermind, I was fooled by the name.